content/influxdb/v2.1/reference/syntax/line-protocol.md

```diff
@@ -82,103 +113,133 @@ _**Key data type:** [String](#string)_
 _**Value data type:** [Float](#float) | [Integer](#integer) | [UInteger](#uinteger) | [String](#string) | [Boolean](#boolean)_
 
 {{% note %}}
-_Always double quote string field values. More on quotes [below](#quotes)._
 
-```sh
-measurementName fieldKey="field string value" 1556813561098000000
+Always double quote string field values. Learn more about using [quotes in line protocol](#quotes).
+
+```py
+measurementName fieldKey="string field value" 1556813561098000000
+measurementName fieldKey="\"quoted words\" in a string field value" 1556813561098000000
 ```
+
 {{% /note %}}
 
+To learn more about designing efficient fields, see [best practices for schema design](/influxdb/v2.1/write-data/best-practices/schema-design/).
+
 ### Timestamp
+
 _**Optional**_ –
-The [unix timestamp](/influxdb/v2.1/reference/glossary/#unix-timestamp) for the data point.
+Unix timestamp for the data point.
 InfluxDB accepts one timestamp per point.
-If no timestamp is provided, InfluxDB uses the system time (UTC) of its host machine.
+
+If no timestamp is provided, InfluxDB uses the system time (UTC) of its host machine. To ensure a data point includes the time a metric is observed (not received by InfluxDB), include the timestamp.
+
+Nanoseconds is the default precision for timestamps. If your timestamps are not in nanoseconds, specify the precision when [writing the data to InfluxDB](/influxdb/v2.1/write-data/#timestamp-precision).
 
 _**Data type:** [Unix timestamp](#unix-timestamp)_
 
-{{% note %}}
-#### Important notes about timestamps
-- To ensure a data point includes the time a metric is observed (not received by InfluxDB),
-  include the timestamp.
-- If your timestamps are not in nanoseconds, specify the precision of your timestamps
-  when [writing the data to InfluxDB](/influxdb/v2.1/write-data/#timestamp-precision).
-{{% /note %}}
+#### Example
+
+```py
+myMeasurementName fieldKey="fieldValue" 1556813561098000000
+                                        -------------------
+                                                  |
+                                              Timestamp
+```
 
 ### Whitespace
+
 Whitespace in line protocol determines how InfluxDB interprets the data point.
-The **first unescaped space** delimits the measurement and the tag set from the field set.
-The **second unescaped space** delimits the field set from the timestamp.
+Allowed whitespace characters are regular spaces (` `) and carriage-returns (`\r`).
```

Comment:
`\r` is only allowed just before a newline - it's not allowed as whitespace otherwise.